tests/conftest.py

```diff
@@ -28,29 +44,125 @@ def bucket_cleanup():
     return bucket_manager
 
 
-@pytest.fixture
-def unique_resource_name(request):
+@pytest.fixture(scope="session")
+def setup_nsfs_server_tls_cert():
     """
-    Generates a unique resource name with the given prefix
+    Configure the NSFS server TLS certification and download the certificate
+    in a local file.
+
+
+
+    Args:
+        config_root (str): The path to the configuration root directory.
+
+    Returns:
+        str: The path to the downloaded certificate file.
```

Comment:
can we remove this as we are not passing any args nor returning anything

Reply:
done in https://github.com/red-hat-storage/noobaa-sa-ci/pull/11/commits/bdf79c5c8a8f0626f87f6e83694334e53086fdbf